assets/js/blocks/reviews/editor-block.js

```diff
@@ -69,9 +56,7 @@ EditorBlock.propTypes = {
 	 * The attributes for this block.
 	 */
 	attributes: PropTypes.object.isRequired,
-	/**
-	 * From withComponentId.
-	 */
+	// from withComponentId
```

Comment:
Just a question, why do we use a different comment format just for this item?

Reply:
We are using this format for most props added by HOCs, so I tried to keep it consistent:
<img src="https://user-images.githubusercontent.com/3616980/63600751-6e854380-c5c4-11e9-9a4e-30c6ea328ce7.png" alt="Screenshot" width="320" />

I don't know what was the rationale when that decision was taken. An idea is so that this kind of comments are skipped if some day we use JSDoc or similar tools (JSDoc takes `/** */` but ignores `//`).